examples/common/text_data.py

```diff
@@ -179,6 +164,7 @@ def build_text_dataloader(cfg: DictConfig, device_batch_size: int):
         shuffle_seed=cfg.dataset.get('shuffle_seed', None),
         num_canonical_nodes=cfg.dataset.get('num_canonical_nodes', 128),
         batch_size=device_batch_size)
+    print(f'num_samples: {len(dataset)}')
```

Comment:
```suggestion
```
This was a bad idea on my part, because len(dataset) will be 1/world_size. Would rather not import `dist` here unnecessarily, so recommending removal.